### PR TITLE
html-forms | Cristofer Martinez

### PR DESCRIPTION
**Pull request by CristoferMar**
html-forms

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            <option value="interesting">Because it's interesting</option>
+            <option value="fun">Because it's fun</option>
+            <option value="powerful">Because it's powerful</option>
+          </select>
+        </div>
+        Favorite Lanuage (choose one)
+        <div>
+          <input type="radio" name="lang" value="html">HTML
+          <input type="radio" name="lang" value="css">CSS
+          <input type="radio" name="lang" value="javascript">JavaScript
```
Comment (codingcodymiller) [inline html-forms/index.html:25]: you should put the text for each option here in a `label` element. Make sure you set up a properly configured `for` attribute!

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        </div>
+        Would you like to sign up for more information?
+        <div>
+          <input type="checkbox" name="subscribe"> Sign me up!
+        </div>
+        Comments
```
Comment (codingcodymiller) [inline html-forms/index.html:31]: you should put this in a `label` element. Make sure you set up a properly configured `for` attribute!

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  <head>
+    <title>html-forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="GET">
+      <div>
+        Email:
+        <div><input type="text" name="email"></div>
+        Why is coding cool?
```
[the remark this answers is not in the document]
Comment (CristoferMar) [reply html-forms/index.html:12]: I've added the following code to "Why is coding cool?":
`<lable for="whycool">Why is coding cool?</lable>`

And I've tried adding the same ID or name to the selector element as well as into each option. Neither of them seems to allow to autoclick into the dropdown

Currently, I have it added to the select element:
`<select name="whycool" id="whycool">`

**Comment by CristoferMar** [comment]
Updated typo
html-forms

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**